src/balance.rs

```diff
@@ -105,16 +107,19 @@ where
 
         let cheques = self.cheques.borrow();
         let mut purse = NegativeImbalanceOf::<T>::zero();
-        let mut depts = Vec::<(&AccountIdOf<T>, BalanceOf<T>)>::new();
+        let mut depts = Vec::<(AccountIdOf<T>, BalanceOf<T>)>::new();
 
         // Calculate balance differences and withdraw negative ones from user's accounts.
         for (account, balance) in cheques.iter() {
             let true_balance = self.initial_state.get(account).unwrap_or(&zero);
+            // let mut ref_acc: &[u8] = account;
+            // let account_id = T::AccountId::decode(&mut ref_acc)
```

Comment:
Do we want to merge these? :male-detective:

Reply:
Of course not! :laughing:
Done :+1: